src/modules/auth/controller.js

```diff
@@ -28,6 +31,34 @@ const tryAuth = async (req, res, next) => {
   }
 };
 
+const refreshToken = async (req, res, next) => {
+  try {
+    const user = userService.getUser({ id: req.user.id });
```

Comment:
Actually there is a `findById` method in sequelize
So you can add similar method to user service and use that method here